File: FizzBuzz.php
<DOCTYPE! html>
<html>
 <head>
 <?php
    for($i = 1; $i <= 100; $i++) {
        if($i % 3 == 0) {
          echo "Fizz";
          }

        elseif($i % 5 == 0) {
          echo "Buzz";
          }

        elseif($i % 3 == 0 and $i % 5 == 0) {
          echo "FizzBuzz";
          }

        else {
          echo "<h6>" . $i . "</h6>";
          }
        }
    ?>
 </head>
<html>

File: fizzbuzz.py
def fizzbuzz():
    for i in range(1, 100):
        if i % 3 == 0:
            print("Fizz\n")
        elif i % 5 == 0:
            print("Buzz\n")
        elif i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz\n")
        else:
            print(i)
            print("\n")

def main():
    fizzbuzz()
